src/components/Sidebar.astro

```diff
@@ -0,0 +1,42 @@
+---
+import type { Props } from '@astrojs/starlight/props';
+
+import MobileMenuFooter from '@astrojs/starlight/components/MobileMenuFooter.astro';
+import SidebarSublist from '@astrojs/starlight/components/SidebarSublist.astro';
+
+const { sidebar } = Astro.props;
+---
+
+<>
+<SidebarSublist sublist={sidebar} />
+<div class="md:sl-hidden">
+	<MobileMenuFooter {...Astro.props} />
+</div>
+<div class="bottom-links">
+  <a class="netlify-link" href="https://netlify.com">
+		<span>This site is powered by Netlify</span>
+	</a>
+</div>
+</>
+
+<style>
+
+  .netlify-link {
+		font-size: var(--sl-text-lg);
+		font-weight: 600;
+		color: var(--sl-color-white);
+    text-decoration: none;
+  }
+
+  .bottom-links {
+    bottom: 2rem;
+    text-align: center;
+    margin: auto;
+    width: 100%;
```

Comment:
This somehow leads to an overflow of the text so that the sidebar gets scrollable (on large screens). Can be seen here: https://starlight-migration--openpodcastapi-test-deploy.netlify.app/specs/

```suggestion
```

If this has no unwanted side effects, removing the `width: 100%` should do the trick.